Description: fix FTBFS with glib2.0 >=2.66.3
 Glib has changed and broke includes from CPP/C context.
 This needs to be resolved for real, but until then we can hard set
 the version we want to avoid the FTFBS.
Forwarded: https://github.com/vmware/open-vm-tools/issues/500
Author: Christian Ehrhardt <christian.ehrhardt@canonical.com>
Origin: https://bugs.launchpad.net/ubuntu/+source/open-vm-tools/+bug/1916705/comments/7
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1916705
Last-Update: 2021-03-08
--- a/open-vm-tools/configure.ac
+++ b/open-vm-tools/configure.ac
@@ -503,6 +503,10 @@ AC_CHECK_PROG([have_genmarshal],
               [yes],
               [no])
 
+# FTFBS workaround, see LP: #1916705
+AC_DEFINE(GLIB_VERSION_MIN_REQUIRED, GLIB_VERSION_2_34, [Ignore post 2.34 deprecations])
+AC_DEFINE(GLIB_VERSION_MAX_ALLOWED, GLIB_VERSION_2_34, [Prevent post 2.34 APIs])
+
 if test "$have_genmarshal" != "yes"; then
    AC_MSG_ERROR([glib-genmarshal is required; make sure it is available in your path.])
 fi
